packages/alfa-style/src/property/display.ts

```diff
@@ -14,11 +17,15 @@ declare module "../property" {
 /**
  * @internal
  */
-export type Specified =
-  | readonly [Specified.Internal]
-  | readonly [Specified.Box]
-  | readonly [Specified.Outside, Specified.Inside]
-  | readonly [Specified.Outside, Specified.Inside, Specified.ListItem];
+export type Specified = Tuple<
+  | [
+      outside: Specified.Outside,
+      inside: Specified.Inside,
+      listitem?: Specified.ListItem
+    ]
+  | [outside: Specified.Internal, inside: Specified.Inside]
+  | [Specified.Box]
+>;
```

Comment:
The alternate on the parameter is less precise and I find it more confusing than putting it outside of `Tuple`.
```suggestion
export type Specified = Tuple<[
      outside: Specified.Outside,
      inside: Specified.Inside,
      listitem?: Specified.ListItem
    ]>
  | Tuple<[outside: Specified.Internal, inside: Specified.Inside]>
  | Tuple<[Specified.Box]>;
```

Reply:
I don't agree. Splitting the type like this is only needed if it should be possible to later narrow the union to a subset of its individual members, which isn't the case here. It's variations of the _same_ `display` value.

Reply:
In that very case, for me it's more of a legibility thing.

I immediately read `Tuple<[A, B]> | Tuple<[C, D]>` as "either (a, b) or (c, d)" while `Tuple<[A, B] | [C, D]>` makes me think in order to get that it cannot be (a, d) nor (c, b) nor some other weird combination. The outer alternative feels clearer to indicate that it's "a tuple like this or a tuple like that" while the inner alternative looks like it could possibly be a way to mix the arguments. Of course, it is different from `Tuple<[A | C, B | D]>` but I need more thinking to reach the same conclusion (and it also looks a bit like `Tuple<[A | B, C | D]>` with just slightly different separators).